novaagent/libs/centos.py

```diff
@@ -168,11 +174,14 @@ def _os_defaults_network_manager():
         :rtype: bool
         :return: has network manager only, not network scripts
         """
-        dist = distro.name()
+        dist = distro.id()
+        server_os_version = ServerOS.version_tuple()
 
-        if dist in ['rhel', 'centos'] and ServerOS.version_tuple() >= (7, 6):
+        log.info("Linux Distribution Detected: {0} Version {1}".format(
+            dist, ServerOS.version_tuple()))
+        if dist in ['rhel', 'centos'] and server_os_version >= (8, 0):
             return True
-        if dist == 'fedora' and ServerOS.version_tuple() >= (29,):
+        if dist == 'fedora' and server_os_version >= (29, 0):
```

Comment:
There is no such thing as Fedora 29.0, this doesn't make sense.

Reply:
Rolled this change back to show the error.